sdk/appcenter-auth/src/test/java/com/microsoft/appcenter/auth/AuthTest.java

```diff
@@ -284,17 +304,45 @@ private void testInvalidConfig(JSONObject jsonConfig) throws Exception {
         Auth auth = Auth.getInstance();
         start(auth);
 
+        /* Mock public client application. */
+        mockMsalPublicClientApplication();
+
+        /* When we get a payload valid for AppCenter fields but invalid for msal ones. */
```

Comment:
```suggestion
        /* When we get a valid payload for App Center fields but an invalid payload for Microsoft Authentication Library (MSAL) ones. */
```

Suggested an edit to:
- reverse the order of adjective and noun 
- include a space for “App Center”
- capitalize [MSAL](https://docs.microsoft.com/azure/active-directory/develop/msal-overview) as a proper noun, and explain and introduce the abbreviation.